wordpress-mobile/WordPress-Android
# Lint: Fix Jetpack Lint Errors

This PR:
1. Fixes all the `jetpack` only, dimension related errors, so that when running `./gradlew lintJetpackVanillaRelease`, Lint successfully builds.
2. Suppress the `jetpack` only, dimension related warnings, so that both, the `wordpress` and `jetpack` Lint reports output the same results, in terms of total and individual number of warnings. Those number are currently `339` for `wordpress` and `336` for `jetpack`, `3` diff. The current discrepancy is due to 2 `editor` lib and 1 `story` submodule overdraw related Lint warnings, which weren't fixed, at this specific time, and on purpose.

To test:
1. Pull the branch locally.
2. Run ` ./gradlew lintWordpressVanillaRelease` and verify everything works as expected. The task should complete successfully.
3. Go to `WordPress/build/outputs` folder and open the `lint-results-wordpressVanillaRelease.html` report. Note the number.
4. Run ` ./gradlew lintJetpackVanillaRelease` and verify everything works as expected. The task should complete successfully.
5. Go to `WordPress/build/outputs` folder and open the `lint-results-jetpackVanillaRelease.html` report. Note the number.
6. Check CI and verify everything works as expected.
7. Note that CI only trigger Lint for `wordpress`. This is expected. As such, you can only compare CI with your local `wordpress` report.

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I run `Lint` in both, `debug` and `release` build types, as well as both, `wordPress` and `jetpack` dimensions to make sure that this change is not affecting the lint report and the actual reported issues, both `errors` and `warnings`.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
